index.html

```diff
@@ -192,10 +211,11 @@ <h2 class="aboutmeintro">About me</h2>
     </section>
     
       <div class="footer">
-      <form class="form"action="https://formspree.io/f/mjvzwzlb" method="post">
         <h2 class="footer-intro">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
```

Comment:
[ ] kindly fix the text position in the desktop version, so it looks like the Figma design 🤓.
your text : 
![histext](https://user-images.githubusercontent.com/62156607/193045773-b6c30821-2dfc-4ab9-b2a2-54b0f038feb2.PNG)

The Figma design text : 
![textdesign](https://user-images.githubusercontent.com/62156607/193045817-b67263f1-6a23-4fe3-8770-3fd4662eefe7.PNG)